tests/nightly/test_large_array.py

```diff
@@ -185,7 +186,36 @@ def test_pick():
     b = mx.nd.ones(shape=(256*35,))
     res = mx.nd.pick(a,b)
     assert res.shape == b.shape
-
+    
+def test_depthtospace():
+    def f(x, blocksize):
```

Comment:
What does this method do? Use a meaningful name.

Reply:
I have updated the names of the function